BIDLicenses.js

```diff
@@ -117,7 +117,7 @@ const checkCommunityLicense = async (licenseKey, communityId, serviceUrl, myKeyP
     const requestId = JSON.stringify({
         ts: Math.round(new Date().getTime() / 1000),
         appid: senderId,
-        uuid: requestUID
+        uuid: requestUID.uuid
```

Comment:
We should copy over all the other attributes of the requestUID instead of just the requireUID.uuid